File: rb/utility/make_context.rb
# KonkanRailwayLivePosition SDK utility: make_context
require_relative '../core/context'
module KonkanRailwayLivePositionUtilities
  MakeContext = ->(ctxmap, basectx) {
    KonkanRailwayLivePositionContext.new(ctxmap, basectx)
  }
end

File: py/konkanrailwayliveposition_sdk/utility/make_context.py
# KonkanRailwayLivePosition SDK utility: make_context

from konkanrailwayliveposition_sdk.core.context import KonkanRailwayLivePositionContext


def make_context_util(ctxmap, basectx):
    return KonkanRailwayLivePositionContext(ctxmap, basectx)
